.cspell.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
+  "version": "0.2",
+  "ignorePaths": ["**/*.json", "**/*.css", "node_modules", "**/*.log", "./src/adapters/supabase/**/**.ts"],
+  "useGitignore": true,
+  "language": "en",
+  "words": ["Nektos", "dataurl", "devpool", "outdir", "servedir", "Supabase", "SUPABASE", "typebox", "ubiquibot", "Smee", "buidling"],
```

Comment:
Why is `buidling` there?

Reply:
I chopped the readme a little before I pushed, I had planned to adjust it further with the answers to those questions above. But `buidling` was in the readme previously which is a web3 reference